### ISSUE TITLE
(Feature Request) Ability to Encode 4K HDR with ffmpeg

### ISSUE THREAD

**Issue by Iam2smart** [Unmanic/unmanic#271]
I was wondering if you could add the ability to be able to encode 4K HDR movies with ffmpeg to make movies smaller. Thanks

**Comment by Josh5** [comment]
encode it into what?

**Comment by Iam2smart** [comment]
Just the ability to re encode to h265 again but make it smaller I guess if that makes sense

**Comment by Josh5** [comment]
No. You can already transcode something to h265. And you can already configure your transcodes to be in main10 with an HDR configuration. So I'm not sure what you are asking.

**Comment by robinsj-zz** [comment]
I assume he’s meaning to take a full 4k video that is already h265, and it encode it to bring the size down.  For example, some 4K tv show episodes are 8-10gbs, and could easily be brought down to a 2gb range with little quality loss.

**Comment by Josh5** [comment]
> I assume he’s meaning to take a full 4k video that is already h265, and encode it to bring the size down.  For example, some 4K tv show episodes are 8-10gbs, and could easily be brought down to a 2gb range with little quality loss.

If this is what he is meaning, would this also involve reducing the resolution? Or just increasing the compression?

**Comment by Iam2smart** [comment]
Increasing the compression is what I meant. Sorry if it was confusing

**Comment by Iam2smart** [comment]
Also being able to select which 4K movies or shows would be helpful
